.github/workflows/install_dependencies_ubuntu.sh

```diff
@@ -9,6 +9,11 @@ apt-get install -y \
     dejagnu \
     iverilog
 
-if [ `lsb_release -rs` != 16.04 ]; then
+REL = `lsb_release -rs | tr -d .`
+if [ $REL -ge 1804 ]; then
```

Comment:
This should be consistent with `build.yaml`, which uses the condition `$REL -le 1604` to decide to use `gold`.  So I would expect to use `$REL -gt 1604` here, or else use `$REL -lt 1804` there.

Reply:
> I see that the Ubuntu 18.04 and 20.04 checks failed -- did something here break them?

I don't know what's going on here yet. I'm waiting for #7 to land before I can investigate further.

Reply:
> This should be consistent with `build.yaml`, which uses the condition `$REL -le 1604` to decide to use `gold`. So I would expect to use `$REL -gt 1604` here, or else use `$REL -lt 1804` there.

That's a good point, and in fact I suspect that may have caused my problem. TBH whether we use lld or gold is slightly arbitrary. At our scale, they're pretty much as fast as each other, but it might be good to use all three linkers (ld.bfd, ld.gold, and ld.lld) to get test coverage. I don't think we're doing anything even remotely exotic with the linker, so I'd be really surprised if we run into linker-specific issues, but it wouldn't hurt.

Reply:
You pushed a new commit, but this mismatch still exists: in one place you compare against 1604 and in the other place you compare against 1804.  If the value were ever 1605 to 1803, things would fail.  Maybe that will never happen, but we might as compare against the same number in both places.

Separately, it looks like the CI is now getting past the point where it failed before, so thats' good.

Reply:
Argh. You're right. Too much branch juggling on different machines.

I also noticed that, somehow, my branch managed to have `export CXXFLAGS="-fuse-ld=gold -O0"` in it. It shouldn't, because we set the linker using `export LDFLAGS="-Wl,-fuse-ld=$LINKER"`, but I can't see how that change is on my branch but not on master.